### PR TITLE
Add USDC

### PR DESCRIPTION
**Pull request by adr-sk**
Adding USD Coin (USDC) `gno.land/r/demo/usdc` .

This is a GRC-20 token to be treated as a stablecoin in testnets, issued only for testing purposes.

### CODE REVIEW

**Review by r3v4s — CHANGES_REQUESTED** [review]
add usdc data in `portal-loop.json` too.

File: grc20/assets.json [hunk 1]
```diff
@@ -97,5 +97,14 @@
...
+  },
+  {
+  "name": "Usd Coin",
+  "pkg_path": "gno.land/r/demo/usdc",
+  "symbol": "USDC",
+  "decimals": 6,
+  "description": "USDC is a GRC20 stablecoin issued solely for testing purposes.",
+  "website_url": "https://beta.gnoswap.io",
+  "image": "/grc20/images/gno_land_r_demo_usdc.svg"
+}
```
Comment (jinoosss) [inline grc20/assets.json]: Check the indentation. 

```suggestion
    "name": "Usd Coin",
    "pkg_path": "gno.land/r/demo/usdc",
    "symbol": "USDC",
    "decimals": 6,
    "description": "USDC is a GRC20 stablecoin issued solely for testing purposes.",
    "website_url": "https://beta.gnoswap.io",
    "image": "/grc20/images/gno_land_r_demo_usdc.svg"
  }
```

File: grc20/portal-loop.json [hunk 2]
```diff
@@ -78,5 +78,15 @@
...
+  },
+  {
+  "name": "Usd Coin",
+  "pkg_path": "gno.land/r/demo/usdc",
+  "symbol": "USDC",
+  "decimals": 6,
+  "chain_id": "portal-loop",
+  "description": "USDC is a GRC20 stablecoin issued solely for testing purposes.",
+  "website_url": "https://beta.gnoswap.io",
+  "image": "/grc20/images/gno_land_r_demo_usdc.svg"
```
Comment (jinoosss) [inline grc20/portal-loop.json]: Check the indentation. 

```suggestion
    "name": "Usd Coin",
    "pkg_path": "gno.land/r/demo/usdc",
    "symbol": "USDC",
    "decimals": 6,
    "chain_id": "portal-loop",
    "description": "USDC is a GRC20 stablecoin issued solely for testing purposes.",
    "website_url": "https://beta.gnoswap.io",
    "image": "/grc20/images/gno_land_r_demo_usdc.svg"
```

**Review by jinoosss — COMMENTED** [review]
Just fix the indentation.

**Review by jinoosss — APPROVED** [review]
LGTM !

**Review by r3v4s — APPROVED** [review]

**Pull request merged**